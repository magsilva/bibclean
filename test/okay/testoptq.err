%% testoptq.bib:79:Final period/dot/fullstop is unexpected, and often wrong, in BibTeX title ``"Baidoties no ekonomikas recesijas, kas skartu visu pasauli, amerik{\=a}{\c{n}}i izv{\=e}las pl{\=a}noto taup{\=\i}{\v{s}}anu atcelt l{\=\i}dz febru{\=a}rim. Mekl{\=e}s citu br{\=\i}numn{\=u}ji{\c{n}}u."''.
