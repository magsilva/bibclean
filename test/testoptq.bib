@Misc{x,
    author = "H\aa kon Sza\ss-\O berg and Yngve {\"{O}}rstad and
                 Hanne \'{A}ngel",
}

@Misc{x,
    author = "Rezs\H o Tarj\'an",
}

@Misc{x,
    author = "Neumann{ }J{\'a}nos and Wigner{ }Jen{\H{o}}",
}

@Misc{x,
    note = "Allegedly written by {H\aa kon Sza\ss-\O berg} and
                 Yngve {\"{O}}rstad and Hanne \'{A}ngel.",
}

@Misc{x,
    title = "The fraternity $\Sigma \Phi \Delta$",
}

@Misc{x,
    title = "Houses in \AE r\o sk\o bing on \AE r\o     ",
}

@Misc{x,
    title = "Huse i \AE r\o sk\o bing p\aa\ \AE r\o     , Danmark",
}

@Misc{x,
    title = "Huse i \AE r\o sk\o bing p\aardvark\ \AE r\o     , Danmark",
}

@Misc{x,
    title =        "A Geometriai {T}{\'e}rfogalom Fejl{\H{o}}d{\'e}se: {A}
                   geometriai fogalmak fejl d{\'e}se
                   {P}{\"u}thagoraszt{\'o}l Hilbertig {\'e}s Einsteinig
                   [{The} geometric concept of space development: The
                   development of geometric concepts from {Pythagoras} to
                   {Hilbert} and {Einstein}]",
  bibsource =    "http://www.math.utah.edu/pub/bibnet/authors/l/lanczos-cornelius.bib",
}

@Misc{x,
  title =        "Az id\H ot{\"{u}}kr{\"{o}}z\'es a
                 kvantummechanik\'aban. ({Hungarian}) [{Mirroring}
                 time in quantum mechanics]",
  bibsource =    "http://www.math.utah.edu/pub/bibnet/authors/w/wigner-eugene.bib",
}

@Misc{x,
  title =        "Keser\H u \'es vigasztal\'o eml\'ekez\'es.
                 ({Hungarian}) [{Bitter} memories and comforting ones]",
  bibsource =    "http://www.math.utah.edu/pub/bibnet/authors/w/wigner-eugene.bib",
}

@Misc{x,
  title =        "A tudom\'any n{\"o}veked\'ese --- kedvez\H o
                 kil\'at\'asok \'es v\'arhat{\'o} vesz\'elyek.
                 ({Hungarian}) [{The} growth of science and a positive
                 outlook: expected hazards]",
  bibsource =    "http://www.math.utah.edu/pub/bibnet/authors/w/wigner-eugene.bib",
}

@Misc{x,
  title =        "\ldots{} Nincs m\'eg \'atfog\'o elm\'elet
                 [El{\"o}ad\'as az {ELTE} {\"u}nnepi
                 {{\"u}l\'es\'en, 1987}. november 17-\'en:
                 Videofelv\'etelr{\"o}l \'at{\'\i}rta {Marx
                 Gy{\"o}rgy}]. ({Hungarian}) [{There} is no
                 comprehensive theory [{Lecture} at the celebration of
                 {E{\"o}tv{\"o}s}, {17 November 1987}: Video recordings
                 transcribed by {George Marx}]]",
  bibsource =    "http://www.math.utah.edu/pub/bibnet/authors/w/wigner-eugene.bib",
}

@Misc{x,
 title = "Baidoties no ekonomikas recesijas, kas skartu visu
          pasauli, amerik\=a\c ni izv{\=e}las pl\=anoto
          taup\=\i\v sanu atcelt l\=\i dz
          febru\=arim. Mekl{\=e}s citu br\=\i numn\=uji\c nu.",
 URL = "http://news.lv/library/regional",
}
