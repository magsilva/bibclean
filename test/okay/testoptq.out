@Misc{x,
  author =       "H{\aa}kon Sza{\ss}-{\O}berg and Yngve {\"O}rstad and
                 Hanne {\'A}ngel",
}

@Misc{x,
  author =       "Rezs{\H{o}} Tarj{\'a}n",
}

@Misc{x,
  author =       "Neumann{ }J{\'a}nos and Wigner{ }Jen{\H{o}}",
}

@Misc{x,
  note =         "Allegedly written by {H{\aa}kon Sza{\ss}-{\O}berg} and
                 Yngve {\"O}rstad and Hanne {\'A}ngel.",
}

@Misc{x,
  title =        "The fraternity {$\Sigma \Phi \Delta$}",
}

@Misc{x,
  title =        "Houses in {\AE}r{\o}sk{\o}bing on {\AE}r{\o}",
}

@Misc{x,
  title =        "Huse i {\AE}r{\o}sk{\o}bing p{\aa}\ {\AE}r{\o},
                 Danmark",
}

@Misc{x,
  title =        "Huse i {\AE}r{\o}sk{\o}bing p\aardvark\ {\AE}r{\o},
                 Danmark",
}

@Misc{x,
  title =        "A Geometriai {T}{\'e}rfogalom Fejl{\H{o}}d{\'e}se: {A}
                 geometriai fogalmak fejl d{\'e}se
                 {P}{\"u}thagoraszt{\'o}l Hilbertig {\'e}s Einsteinig
                 [{The} geometric concept of space development: The
                 development of geometric concepts from {Pythagoras} to
                 {Hilbert} and {Einstein}]",
  bibsource =    "http://www.math.utah.edu/pub/bibnet/authors/l/lanczos-cornelius.bib",
}

@Misc{x,
  title =        "Az id{\H{o}}t{\"u}kr{\"o}z{\'e}s a
                 kvantummechanik{\'a}ban. ({Hungarian}) [{Mirroring}
                 time in quantum mechanics]",
  bibsource =    "http://www.math.utah.edu/pub/bibnet/authors/w/wigner-eugene.bib",
}

@Misc{x,
  title =        "Keser{\H{u}} {\'e}s vigasztal{\'o} eml{\'e}kez{\'e}s.
                 ({Hungarian}) [{Bitter} memories and comforting ones]",
  bibsource =    "http://www.math.utah.edu/pub/bibnet/authors/w/wigner-eugene.bib",
}

@Misc{x,
  title =        "A tudom{\'a}ny n{\"o}veked{\'e}se --- kedvez{\H{o}}
                 kil{\'a}t{\'a}sok {\'e}s v{\'a}rhat{\'o} vesz{\'e}lyek.
                 ({Hungarian}) [{The} growth of science and a positive
                 outlook: expected hazards]",
  bibsource =    "http://www.math.utah.edu/pub/bibnet/authors/w/wigner-eugene.bib",
}

@Misc{x,
  title =        "\ldots{} Nincs m{\'e}g {\'a}tfog{\'o} elm{\'e}let
                 [El{\"o}ad{\'a}s az {ELTE} {\"u}nnepi
                 {{\"u}l{\'e}s{\'e}n, 1987}. november 17-{\'e}n:
                 Videofelv{\'e}telr{\"o}l {\'a}t{\'\i}rta {Marx
                 Gy{\"o}rgy}]. ({Hungarian}) [{There} is no
                 comprehensive theory [{Lecture} at the celebration of
                 {E{\"o}tv{\"o}s}, {17 November 1987}: Video recordings
                 transcribed by {George Marx}]]",
  bibsource =    "http://www.math.utah.edu/pub/bibnet/authors/w/wigner-eugene.bib",
}

@Misc{x,
  title =        "Baidoties no ekonomikas recesijas, kas skartu visu
                 pasauli, amerik{\=a}{\c{n}}i izv{\=e}las pl{\=a}noto
                 taup{\=\i}{\v{s}}anu atcelt l{\=\i}dz febru{\=a}rim.
                 Mekl{\=e}s citu br{\=\i}numn{\=u}ji{\c{n}}u.",
  URL =          "http://news.lv/library/regional",
}
